src/main/java/carRacing/view/ResultView.java

```diff
@@ -30,8 +30,7 @@ public static void printRacingCarName(List<RacingCar> records){
     }
 
     private static void printPlayCarName(RacingCar car){
-
-        System.out.println(car.NameOfCar() + " : " + "-"
+        System.out.printf("%s : %s%n", car.name(), "-"
                 .repeat(Math.max(0, car.position())));
```

Comment:
```suggestion
                .repeat(car.position()));
```
포지션만큼 반복을 돌리는 데 max 함수는 필요없을 것 같아요